R/namespace.R

```diff
@@ -238,6 +238,15 @@ roxy_tag_parse.roxy_tag_importFrom <- function(x) {
 }
 #' @export
 roxy_tag_ns.roxy_tag_importFrom <- function(x, block, env, import_only = FALSE) {
+  pkg <- x$val[1L]
+  if (requireNamespace(pkg, quietly = TRUE)) {
+    importing <- x$val[-1L]
+    unknown_idx <- !importing %in% getNamespaceExports(pkg)
```

Comment:
two issues with this PR in retrospect as surfaced by #1570:

 1. `importing` can be like `"%>%"` or `` `%>%` `` or `'%>%'` but `getNamespaceExports()` will always just be the plain symbol (`%>%`)
 2. `importing[!unknown_idx]` might drop everything --> output becomes `importFrom($PKG,)`

How would you recommend approaching these?

 1. Is a bit tricky because we have to normalize the quotes, I guess we can do `gsub("^(`|'|\")(.*)\\1$", "\\1", importing)`. This could be a helper alongside [`has_quotes()`](https://github.com/r-lib/roxygen2/blob/767ec1b1c4dc43b4245ed4ee2334445968768f17/R/utils.R#L181) Initially, I was worried about collision where you try and import a really weirdly-named object like `as.name("'%>%'")`, but I guess {roxygen2} has always made that difficult. And a quick search doesn't turn up any usage of this anyway: https://github.com/search?q=lang%3AR+%2F%5Cn%5Cs*%28%22%27.*%27%22%7C%27%22.*%22%27%7C%60%22.*%22%60%7C%60%27.*%27%60%7C%22%60.*%60%22%7C%27%60.*%60%27%29%5Cs*%3C-%2F+-path%3A.Rd&type=code
 2. Should this method just `return(NULL)` if `all(unknown_idx)`?

Reply:
1. Yeah, I think `strip_quotes()` seems reasonable.
2. Yes, I think sol.

Reply:
thanks! that's basically how I did #1574 so let's continue there.